src/main/java/org/broadinstitute/consent/http/models/tdr/ApprovedUser.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.consent.http.models.tdr;
+
+public class ApprovedUser {
+    private String email;
```

Comment:
Thank you for not perpetuating the @JsonProperty pattern ... I'm intending on fixing this every time I touch the codebase